assets/src/edit-story/utils/textMeasurements.js

```diff
@@ -32,6 +32,7 @@ const MEASURER_STYLES = {
   contain: 'layout paint',
   top: '-9999px',
   left: '-9999px',
+  fontDisplay: 'block',
```

Comment:
`font-display` cannot be applied to the element itself. We have to apply it to the `@font-face`. Not sure exactly where that can be done. But I'd start with `maybeEnqueFont`. Google Fonts API allows us to force `font-display:block` on font face with some query parameter. And maybe that's already done, but we need to check.

Reply:
We just need to make sure that the loaded `@font-face`s definitely do not have `fallback` or `optional`, `swap`. And ideally it'd be `block`, though we can live with `auto` for some time.

Reply:
To clarify: it's possible it's all good already (or at least ok). Let's check it first.

Reply:
Just covered this case in the `maybeEnqueueFontStyle`, but I think that we can (should?) deal with this in the API given the origin of the font source. cc WDYT @miina ?

Reply:
NVM, given this comment https://github.com/google/web-stories-wp/issues/1348#issuecomment-620057259, would be better to stay with a client-side approach to deal with this. Thoughts?

Reply:
It should definitely be all client-side.